recipes/sparkmagic/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sparkmagic" %}
+{% set version = "0.12.1" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "ac33c480b26d6c254b8f34f539a3e010dc00917ab2e182443d40d268351dbad4" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    {{ hash_type }}: {{ hash_val }}
+
+build:
+    number: 0
+    skip: true  # [win]
+    script: python setup.py install
+
+requirements:
+    build:
+        - python
+    run:
+        - python
+        - notebook >=4.2
+        - ipykernel >=4.2.2
+        - ipywidgets >=5.0.0
+        - ipython >=4.0.2
+        - widgetsnbextension
+        - hdijupyterutils
+        - autovizwidget
```

Comment:
There will be a race condition and the packages depending on packages that are not built yet will fail. You can either submit separate PRs or restart the CIs manually later. Are you OK with this?